src/main/java/bookstore/controller/BookController.java

```diff
@@ -25,11 +29,23 @@ public List<BookDto> getAll() {
 
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable Long id) {
-        return bookService.findBookById(id);
+        return bookService.findById(id);
     }
 
     @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
consider adding response statuses for all the endpoints

Reply:
All other made by default, but ok